source/languageHandler.py

```diff
@@ -94,29 +98,34 @@ def getLanguageDescription(language):
 		}.get(language,None)
 	return desc
 
-def getAvailableLanguages():
+def getAvailableLanguages(presentational=False):

```

Comment:
I think alphabetize or sort might be a clearer keyword argument instead of presentational. Furthermore, someone might be interested in getting a sorted list of languages for non presentational purposes.